### PR TITLE
IPAC spectra convert

### PR DESCRIPTION
**Pull request by jolielheureux**

### CODE REVIEW

**Comment by jolielheureux** [comment]
converted data are in issue #308

**Comment by kelle** [comment]
Script looks good but some of the filenames are still a little wonky. Eg, BRI and the question mark in 2M1207334.

As we discussed, use the source from the database for the object name and use that (with the spaces removed) for the filename.

File: scripts/spectra_convert/convert_spectra_to_spec1dfits.py [hunk 1]
```diff
@@ -94,7 +94,7 @@ def convert_to_fits(spectrum_info_all):
     ax.set_xlabel(f"Dispersion ({spec1d.spectral_axis.unit})")
     ax.set_ylabel(f"Flux ({spec1d.flux.unit})")
     plt.title(f"{name} {header['TELESCOP']} {header['INSTRUME']}")
-    plt.show()
```
Comment (kelle) [inline scripts/spectra_convert/convert_spectra_to_spec1dfits.py:97]: Lets generate PNGs. :)

**Review by kelle — APPROVED** [review]
Looks great!  One minor comment about using `strip` instead of `replace`.

**Comment by kelle** [comment]
Please upload the converted files to Issue #308 and I'll upload them to AWS.

**Pull request merged**